templates/files/apps/azure/azuredisk-csi-driver.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: application.giantswarm.io/v1alpha1
+kind: App
+metadata:
+  name: azuredisk-csi-driver
+  namespace: giantswarm
+spec:
+  catalog: control-plane-catalog
+  config:
+    configMap:
+      name: ""
+      namespace: ""
+    secret:
+      name: ""
+      namespace: ""
+  install: {}
+  kubeConfig:
+    context:
+      name: ""
+    inCluster: true
+    secret:
+      name: ""
+      namespace: ""
+  name: azuredisk-csi-driver-app
+  namespace: kube-system
+  namespaceConfig: {}
+  userConfig:
+    configMap:
+      name: ""
+      namespace: ""
+    secret:
+      name: ""
+      namespace: ""
+  version: v1.13.0-gs1
```

Comment:
How come we need custom builds for all these components?

Reply:
it's not custom builds, I'm using the same versioningstrategy as we do for cluster-autoscaler.
I'm using the upstream version, but with `-gs<n>` suffix to allow us to version our helm charts.
Does it make sense?